extensions/dict_SWm/dict_SWm.c

```diff
@@ -0,0 +1,519 @@
+/****************
+ Generated By: fdtool enhancements to diafuzzer
```

Comment:
@pingping-lin   what should be license in this case?

Reply:
It should be Apache 2.  I will update.